node/src/components/block_validator.rs

```diff
@@ -59,6 +63,9 @@ pub enum Event<T, I> {
 /// Tracks whether or not there are deploys still missing and who is interested in the final result.
 #[derive(DataSize, Debug)]
 pub(crate) struct BlockValidationState<T, I> {
+    /// Total running size of the deploys in the block, which is used to validate against the
+    /// chainspec's deploy.max_block_size.
+    deploy_size_running_total: usize,
```

Comment:
Just for my own understanding, can't `usize` end up being a different "type" on different machines/architectures?